packages/store/src/tables/Route.sol

```diff
@@ -13,13 +13,16 @@ import { EncodeArray } from "../tightcoder/EncodeArray.sol";
 import { Schema, SchemaLib } from "../Schema.sol";
 import { PackedCounter, PackedCounterLib } from "../PackedCounter.sol";
 
+uint256 constant _tableId = uint256(keccak256("/tables/Route"));
+uint256 constant RouteTableId = _tableId;
+
 struct Route {
   address addr;
   bytes4 selector;
   uint8 executionMode;
 }
 
-library Route_ {
+library RouteTable {
```

Comment:
I think I missed the naming conversation but did ya'll consider `RouteStore` as a name option?

Reply:
I think it'll be confusing when a table that writes to `Store` is also called `Store`. We already have `World`, which is `Store`. Wouldn't want everything to be a `Store`

Reply:
My thought is that users will mostly be interacting with these autogen files, not the underlying ones (i.e. Store). But I understand if this creates confusion on our end.

"Table" just feels like an overloaded term. Folks with SQL experience will have preconceived notions about what it does/should do, and folks without SQL experience may be a little confused by it the term "table". We have a chance to shed this here.

Reply:
I like the SQL analogy (I have SQL experience, although I'm obviously not a great example).
We even have hooks/indexers. And a planned postgres integration. In solidity it's just a limited key/value store of course.
But I'm starting to think of how foreign keys would fit in here

Reply:
to circle back on this: we all chatted in discord and ended up with removing the suffix altogether on libraries, and adding "data" suffix to structs